cli/util/upload_bin.go

```diff
@@ -61,6 +61,9 @@ const (
 	defaultHealthcheckTimeout = 10 * time.Second
 )
 
+// emptyOID is a zeroed object ID.
+var emptyOID = make([]byte, oidSize)
+
```

Comment:
We have really a lot of duplicating code, it's getting irritating. Create a separate package for FS block storage related functionality, move helper functions to this package, reuse them from both BlockFetchr and uploader.